packages/node-solana/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@subql/node-solana",
+  "version": "0.28.2",
+  "description": "",
+  "author": "Ian He",
+  "license": "Apache-2.0",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "rm -rf dist && tsc -b",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nodemon",
+    "start:prod": "node dist/main"
+  },
+  "homepage": "https://github.com/subquery/subql",
+  "repository": "github:subquery/subql",
+  "bin": {
+    "subql-node-solana": "./bin/run"
+  },
+  "dependencies": {
+    "@apollo/client": "3.5.8",
+    "@nestjs/common": "^8.2.6",
+    "@nestjs/core": "^8.2.6",
+    "@nestjs/event-emitter": "^1.0.0",
+    "@nestjs/platform-express": "^8.2.6",
+    "@nestjs/schedule": "^1.0.2",
+    "@polkadot/api": "7.8.1",
```

Comment:
This can be removed

Reply:
I create a new pull request with node applied changes from node-solana. please check it out! 
https://github.com/subquery/subql-solana/pull/2